element_interface/utils.py

```diff
@@ -134,6 +138,10 @@ def recursive_search(key, dictionary):
         :param dictionary: nested dictionary
         :return a: value of the input argument `key`
     """
+    logger.warn(
```

Comment:
This may be overkill. The function was used in one place, so simply removing should not cause anyone any harm.